flavors.gradle

```diff
@@ -0,0 +1,30 @@
+ext {
```

Comment:
Changed from 3 build types to 2 build types and 2 flavors. makes it simpler to have just 2 release and production logger implementations -- besides it just seems like the better structure for what we're doing with targeting local vs prod server.